articles/connections/azuread-adfs-email-verification.md

```diff
@@ -31,7 +31,9 @@ When the property is set to **Always set `email_verified` to `false`**, users wi
 
 In previous versions, Auth0 always set the `email_verified` field to true in Azure AD and ADFS connections. If you were using Azure AD and ADFS connections in the past, you will have a tenant setting that will override the Connection Setting for **Email Verification** and keep the previous behavior. 
 
-The new tenant setting is in the **Migrations** section, and it's called **Default to 'Email Verification' setting for Azure AD/ADFS connections**.
+The new tenant setting is in the Migrations section of the [Dashboard Advanced Tenant Settings](https://manage.auth0.com/#/tenant/advanced) page , and it is called Default to 'Email Verification' setting for Azure AD/ADFS connections.
+
+![](media/connections/migration-section.png)
```

Comment:
Screenshots generally show the whole window and are taken on a mac with the dropshadow background, but we are starting to move away from that. However, we do require a 1 px wide #cccccc border around all images that don't have a drop shadow. You can see the screenshot guidelines in the contributing.md guide, too.

Reply:
Also, there shouldn't be a space between the screenshot and the border.